### PR TITLE
feature: Basic Draytek Support

### PR DESCRIPTION
**Pull request by InsaneSplash**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x ] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

### CODE REVIEW

File: tests/snmpsim/draytek.snmprec [hunk 1]
```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|DrayTek Corporation
```
Comment (murrant) [inline tests/snmpsim/draytek.snmprec]: Is this the actual sysDescr from the device?

Comment (InsaneSplash) [inline tests/snmpsim/draytek.snmprec]: Yes, 

.1.3.6.1.2.1.1.1.0 = STRING: DrayTek Corporation
.1.3.6.1.2.1.1.2.0 = OID: .1.3.6.1.4.1.7367
.1.3.6.1.2.1.1.3.0 = Timeticks: (63065182) 7 days, 7:10:51.82
.1.3.6.1.2.1.1.4.0 = STRING: info@draytek.com
.1.3.6.1.2.1.1.5.0 = STRING: ippbx

Comment (murrant) [inline tests/snmpsim/draytek.snmprec]: Gotta love those descriptive descriptions :)

Comment (InsaneSplash) [inline tests/snmpsim/draytek.snmprec]: Yeah!,

Why did the build fail though?

**Pull request merged**